internal/server/v1/optimus/service.go

```diff
@@ -46,3 +59,29 @@ func (svc *Service) ListJobs(ctx context.Context, projectName string) ([]*optimu
 
 	return res.JobSpecificationResponses, nil
 }
+
+func (svc *Service) createOptimusClient(ctx context.Context, projectName string) (optimusv1beta1grpc.JobSpecificationServiceClient, error) {
```

Comment:
LGTM 👌 , just need to introduce caching.

we can store the cache in the `Service` field.

Reply:
done